docs/_docs/focus/convert.md

```diff
@@ -57,35 +62,33 @@ The following mapping is assuming you have all amortized cost rows and only comm
-| PricingQuantity            | Quantity                                            | Map UnitOfMeasure using [Pricing units data file](../../_reporting/data/README.md#-pricing-units) and divide Quantity by the PricingBlockSize<sup>2</sup>             |
-| PricingUnit                | UnitOfMeasure                                       | Map using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                                                                   |
+| PricingQuantity            | Quantity / x_PricingBlockSize (lookup)              | Map UnitOfMeasure to x_PricingBlockSize using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)<sup>1</sup>                                   |
+| PricingUnit                | DistinctUnits (lookup)                              | Map UnitOfMeasure to DistinctUnits using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                                    |
 | ProviderName               | `Microsoft`                                         | None                                                                                                                                                                  |
 | PublisherName              | PublisherName                                       | None                                                                                                                                                                  |
 | RegionId                   | focus:RegionName                                    | Lowercase and remove spaces                                                                                                                                           |
-| RegionName                 | ResourceLocation                                    | Map using [Regions data file](../../_reporting/data/README.md#-regions)<sup>3</sup>                                                                                   |
+| RegionName                 | ResourceLocation                                    | Map ResourceLocation to RegionName using [Regions data file](../../_reporting/data/README.md#-regions)<sup>2</sup>                                                    |
 | ResourceId                 | ResourceId                                          | None                                                                                                                                                                  |
 | ResourceName               | ResourceName                                        | None                                                                                                                                                                  |
```

Comment:
It can, but it's not that simple. Nested resources technically have a name that includes all levels in the resource hierarchy. So a SQL db (under a server) would be "MyServer/MyDatabase".

I forgot that this isn't in the MCA dataset today 🤔 Doing the "right" thing is a bit onerous and probably wouldn't be clear here. If I remember correctly from doing this in KQL, it requires a few lines of code. Your approach is simpler, but I hate to rely on regular expressions. What do you think about this:
```suggestion
| ResourceName               | EA: ResourceName<br>MCA: split(ResourceId, "/")[-1] | Azure resource names include multiple levels (e.g., "SqlServerName/SqlDbName"), which requires more processing. This is a simplified approach to only use the last, most-specific segment. |
```

Reply:
Maybe use `length(ResourceId) - 1` as the array index?  I'm not sure if `-1` will be clear.

Reply:
Oh, what about this:
```suggestion
| ResourceName               | EA: ResourceName<br>MCA: last(split(ResourceId, "/")) | Azure resource names include multiple levels (e.g., "SqlServerName/SqlDbName"), which requires more processing. This is a simplified approach to only use the last, most-specific segment. |
```
Not many platforms have a last() function, but it's at least conceptually simpler to understand.

Reply:
I'll go with this for now so we can get these changes into this release. Let me know if you think there's a better way. Happy to find a better alternative.